classes/hazelcast-cache-config.xml

```diff
@@ -17,13 +17,15 @@
 <hazelcast xmlns="http://www.hazelcast.com/schema/config"
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://www.hazelcast.com/schema/config
-           http://www.hazelcast.com/schema/config/hazelcast-config-3.12.xsd">
+           http://www.hazelcast.com/schema/config/hazelcast-config-5.3.xsd">
     <import resource="${openfireHome}/conf/hazelcast-local-config.xml"/>
+    <cluster-name>openfire-cluster-by-hazelcast</cluster-name>
     <properties>
       <property name="hazelcast.logging.type">slf4j</property>
       <property name="hazelcast.operation.call.timeout.millis">30000</property>
     </properties>
-    <management-center enabled="false"/>
+    <jet enabled="true"/>
```

Comment:
Sure,  we can remove this from hazelcast-cache-config.xml by default,  this can be configured on hazelcast-local-config.xml if user need this feature.

Reply:
Please create a new PR with this change, so that we do not loose it!